dask_ml/model_selection/_incremental.py

```diff
@@ -686,10 +745,16 @@ class IncrementalSearchCV(BaseIncrementalSearchCV):
         If specified, training stops when the score does not increase by
         ``tol`` after ``patience`` calls to ``partial_fit``. Off by default.
 
+    fits_per_scores : int, optional, default=1
+        If ``patience`` is used the maximum number of ``partial_fit`` calls
+        between ``score`` calls.
+
     scores_per_fit : int, default 1
         If ``patience`` is used the maximum number of ``partial_fit`` calls
         between ``score`` calls.
 
+        *Deprecated in Dask-ML v1.4.0.*
```

Comment:
You can use the deprecated directive: http://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-deprecated